e2e-tests/src/test/emergency_finalizer.rs

```diff
@@ -0,0 +1,26 @@
+use aleph_client::{
+    pallets::aleph::{AlephApi, AlephSudoApi},
+    waiting::{BlockStatus, WaitingExt},
+    AccountId, AlephKeyPair, AsConnection, Pair, TxStatus,
+};
+
+use crate::config::setup_test;
+
+#[tokio::test]
+async fn set_emergency_finalizer_test() -> anyhow::Result<()> {
+    let config = setup_test();
+    let (finalizer, _seed) = AlephKeyPair::generate();
+    let public: AccountId = finalizer.public().0.into();
+    let root = config.create_root_connection().await;
+    let current_finalizer = root.as_connection().emergency_finalizer(None).await;
+
+    assert!(current_finalizer.is_none());
```

Comment:
nit: perhaps it would be a good idea to check just whether the retrieved one differs from the one we set. This way this test should be "more compatible" with other tests/doesn't assume it was run on a fresh chain. btw it's little bit confusing that a finalizer can be `None` but we can't set it to `None` (not related with this PR).

Reply:
ok done. In my free time I will add option to set finalizer to None